daemon/algod/server.go

```diff
@@ -65,10 +65,19 @@ func (s *Server) Initialize(cfg config.Local) error {
 	// set up node
 	s.log = logging.Base()
 
-	liveLog := fmt.Sprintf("%s/node.log", s.RootPath)
-	archive := fmt.Sprintf("%s/node.archive.log", s.RootPath)
+	liveLog := filepath.Join(s.RootPath, "node.log")
+	archive := filepath.Join(s.RootPath, cfg.LogArchiveName)
 	fmt.Println("Logging to: ", liveLog)
-	logWriter := logging.MakeCyclicFileWriter(liveLog, archive, cfg.LogSizeLimit)
+	var maxLogAge time.Duration
+	var err error
+	if cfg.LogArchiveMaxAge != "" {
+		maxLogAge, err = time.ParseDuration(cfg.LogArchiveMaxAge)
+		if err != nil {
+			s.log.Errorf("invalid config LogArchiveMaxAge: %s", err)
```

Comment:
Can we s.log.Fatal() here since this should not be silently ignored - it's an ill-formed config.json file.  We otherwise log.Fatal() in algod/main.go if we encounter other errors loading the config file (which this would do is were were decoding it as a duration when loading the json).
> log.Fatalf("Cannot load config: %v", err)